Create How-to-deactivate-mods.md (#120)
BeamMP/Docs: docs/en/FAQ/How-to-deactivate-mods.md

> # I need help deactivating my mods!

Reviewer: ```suggestion
# Why do i need to deactivate my mods?
```

> Having Mods besides BeamMP installed or enabled can lead to issues with the operation of BeamMP.

Reviewer: ```suggestion
In BeamMP, the Server you decide to connect to, provides the mods.
```

> When you are not able to disable mods or all mods (e.g. stuck in loading screen), it is likely that atleast one corrupt, broken or incompatible mod in the mod manager.

Reviewer: ```suggestion
They get downloaded and activated automatically upon connecting.
```

> This often leads to BeamMP not functioning properly, even if you have just one additional mod besides BeamMP.

Reviewer: ```suggestion
Having local mods installed and active often leads to BeamMP not functioning properly, even if you have just one additional mod besides BeamMP.
This includes mods placed into the BeamNG/Content folder. As stated by BeamNG in their text file:

!!! warning "Do NOT copy mods into this folder: it can lead to broken mods, slower installation of updates, a broken mod manager, broken Safe Mode and others."
```

## How can I fix this?

> There are 2 options to do this.

Reviewer: ```suggestion
There are 3 options to do this:
1. In your BeamNG main menu, go to Repository, then Mods Manager. Deactivate all mods besides `multiplayerbeammp`
```

Author: What's the 3rd option?

Reviewer: I see the confusion. Ive added a new line and a new option onewith this suggestion

Author: I don't see any change anywhere

> 1. Open your mods folder and delete all mods permanently.

Reviewer: ```suggestion
2. If for some reason you cant deactivate all mods trough the ingame Repo, then open your mods folder and delete all mods permanently. Alternatively you can
```

> 2. Open the BeamNG.Drive userfolder and rename the versioned folder (e.g. 0.34) to for example 0.xx_OLD (0.34_OLD).

Reviewer: ```suggestion
3. Open the BeamNG.Drive userfolder and rename the folder of the latest version (e.g. 0.34) to for example 0.xx_OLD (0.34_OLD).
```

Reviewer: Heres the third option @Protogen187

Author: That's literally the same?

Author: I'd understand if you made the changes to option 2 that you requested to be changes as option 3

Make sure BeamNG.Drive is closed during either process!
After you are done, start BeamNG.Drive via the BeamMP-Launcher and you should have 'multiplayerbeammp' as your only enabled mod available in the repository as well as the button on the Main Menu to enter BeamMP.

## My settings and configs are gone!

If you renamed the userfolder, you forced the game to craete a new, clean userfolder. You may copy the 'settings' and 'vehicles' folder from the folder you renamed (e.g. 0.34_OLD) to the new folder it created.

> Make sure BeamNG.Drive is closed and replace all elements in the location you want to copy the folders to. You should not have all configs and settings as they were before.

Reviewer: ```suggestion
Make sure BeamNG.Drive is closed and replace all elements in the location you want to copy the folders to. You should now have all configs and settings as they were before.
```
